prestodb/client.py

```diff
@@ -422,6 +434,15 @@ def process(self, http_response):
             columns=response.get("columns"),
         )
 
+    @property
+    def http_session(self):
+        return self._http_session
+
+    def get_oauth_token(self):
+        self.credentials.refresh(self.auth_req)
+        return {
+            constants.PRESTO_EXTRA_CREDENTIAL: "%s = %s" % (constants.GCS_CREDENTIALS_OAUTH_TOKEN_KEY, self.credentials.token)
```

Comment:
This line seems to be too long. I noticed that the tool `black` is used at this codebase for formatting. Would you mind formatting your code with `black`?
A reference: https://github.com/prestodb/presto-python-client/commit/d48f458799982909111c76707127835c3836f928.

Reply:
Updated.